### PR TITLE
actions: manifest: check additional metadata in projects

### PR DESCRIPTION
**Pull request by carlescufi**
See https://github.com/zephyrproject-rtos/action-manifest/pull/14

Action test: https://github.com/zephyrproject-rtos/zephyr-testing/pull/291

### CODE REVIEW

**Comment by carlescufi** [comment]
DNM until the action points to a SHA/tag instead of a PR

File: .github/workflows/manifest.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ jobs:
           west init -l . || true
 
       - name: Manifest
-        uses: zephyrproject-rtos/action-manifest@v1.3.1
+        uses: zephyrproject-rtos/action-manifest@pull/14/head
```
Comment (fabiobaltieri) [inline .github/workflows/manifest.yml:29]: `441908916ee486cbf4c0b6d2dbb3fb01d48c83a5`

Comment (fabiobaltieri) [inline .github/workflows/manifest.yml:29]: you can probably push a `v1` tag that points at the lastes `v1.x.x` and the updated it when needed, then you won't have to update it here, I think that's how actions are normally managed. This is fine as well though, you have to create this anyway for testing.

Comment (carlescufi) [reply .github/workflows/manifest.yml]: ah yes, that is a very good idea! Will do this next time.

**Review by fabiobaltieri — APPROVED** [review]

**Review by nashif — APPROVED** [review]

**Pull request merged**